### PR TITLE
Release 4.3.0

### PR DESCRIPTION
**Pull request by jvanderaa**
Release 4.3.0 Prep.

### CODE REVIEW

**Comment by itdependsnetworks** [comment]
tests/test_structured_data_against_parsed_reference_files.py::test_raw_data_against_mock[tests/fortinet/get_system_status/fortinet_get_system_status_7.0.raw] FAILED [ 42%]

**Comment by jvanderaa** [comment]
Working to fix the tests. Not sure how the tests started failing when all of them passed to get merged in.

**Comment by jvanderaa** [comment]
@itdependsnetworks this is now ready.

File: pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "ntc_templates"
-version = "4.0.1"
+version = "4.3.0"
```
Comment (itdependsnetworks) [inline pyproject.toml:3]: How is this changing from 4.0.1 -> 4.3.0?

Comment (jvanderaa) [reply pyproject.toml:3]: That is a great question. The releases on GitHub are lining up, have the appropriate publications to Pypi in the right versions. How the pyproject.toml and GH versions are out of sync is something that I do not have the answer to. Unless maybe something got merged from a 4.0.1 base.

**Review by itdependsnetworks — APPROVED** [review]

**Pull request merged**